### DISCUSSION TITLE
How to make an item/row permanent?

### DISCUSSION THREAD

**Discussion by owzim** [davatorium/rofi#1948] [Q&A]
Is there a way to make an item/row permanent in dmenu mode and not be filtered out?

Thanks!

**Comment by DaveDavenport** [comment]
No this is not possible.

**Reply by owzim** [reply]
@DaveDavenport thanks for the reply. Is it a feasible feature or totally out of scope? Do you need me to provide some points, why this could be useful?

**Reply by DaveDavenport** [reply]
Its an feasible request, I don't think more information is needed.

**Reply by owzim** [reply]
Nice, should I make a feature request then?

**Reply by DaveDavenport** [reply]
implemented.
